pkg/admin/graphql/query.go

```diff
@@ -333,7 +334,7 @@ var query = graphql.NewObject(graphql.ObjectConfig{
 
 				userIDs, err := gqlCtx.UserFacade.GetUsersByStandardAttribute(attributeName, attributeValue)
 				if err != nil {
-					return nil, err
+					return nil, api.ErrGetUsersInvalidArgument.New(err.Error())
```

Comment:
Do not use api.ErrGetUsersInvalidArgument here. It is because the error could be something else, like I/O error. Just `return nil, err` here.

Reply:
moved api invalid error inside coordinator